### ISSUE TITLE
Error when evaluating ConvE model (other models have no issues)

### ISSUE THREAD

**Issue by alexdevmotion** [Accenture/AmpliGraph#189]
### Description
Executing
```
evaluate_performance(test, model=model, filter_triples=X, corrupt_side='s,o')
```
or
```
evaluate_performance(test, model=model, filter_triples=X, use_default_protocol=True)
```

### Actual Behavior
generates the following error
```
...
  File "C:\anaconda3\envs\ampligraph\lib\site-packages\ampligraph\evaluation\protocol.py", line 645, in evaluate_performance
    raise e
  File "C:\anaconda3\envs\ampligraph\lib\site-packages\ampligraph\evaluation\protocol.py", line 633, in evaluate_performance
    ranks = model.get_ranks(dataset_handle)
  File "C:\anaconda3\envs\ampligraph\lib\site-packages\ampligraph\latent_features\models\ConvE.py", line 959, in get_ranks
    eval_protocol = self.eval_config.get('corrupt_side', constants.DEFAULT_PROTOCOL_EVAL)
AttributeError: module 'ampligraph.latent_features.constants' has no attribute 'DEFAULT_PROTOCOL_EVAL'
```

### Expected Behavior
no error

### Steps to Reproduce
```
params = {
            'batches_count': 500,
            'epochs': 300,
            'k': 200,
            'loss': 'bce',
            'loss_params': {'label_smoothing': .1},
            'optimizer_params': {'lr': .0001},
            'embedding_model_params': {'conv_filters': 32, 'conv_kernel_size': 3, 'dropout_embed': 0.2,
                                       'dropout_conv': 0.1, 'dropout_dense': 0.3, 'use_batchnorm': True,
                                       'use_bias': True},
           'verbose': True
}
model = ConvE(**params)
model.fit(any_train_data)
evaluate_performance(test, model=model, filter_triples=X, corrupt_side='s,o')
```
```
>>> ampligraph.__version__
'1.3.0'
```
```
>>> sys.version
'3.6.10 |Anaconda, Inc.| (default, Jan  7 2020, 15:18:16) [MSC v.1916 64 bit (AMD64)]'
```
```
>>> tensorflow.__version__
'1.13.1'
```

**Comment by NicholasMcCarthy** [comment]
Hi @alexdevmotion ,

Thanks for reporting the bug :) 

This has been fixed on branch feature/189, a pull request to develop is pending and will be merged into the master branch shortly after.

**Comment by lukostaz** [comment]
Hello @alexdevmotion we have just released 1.3.1 which includes this fix.
Please go ahead with `pip install --upgrade ampligraph` . 
Thanks for reporting the problem.
